examples/rust/Earthfile

```diff
@@ -1,29 +1,19 @@
-VERSION 0.7
-FROM rust:1.72
-WORKDIR /rustexample
-
-install-chef:
-    RUN cargo install --debug cargo-chef
+VERSION --global-cache 0.7
+IMPORT github.com/earthly/lib/rust AS rust
```

Comment:
It might be good practice to pin this to a tag. It'll help with not breaking people's setup when we change the UDC.